AirCasting/SearchAndFollow/SearchMap/BottomCard.swift

```diff
@@ -0,0 +1,67 @@
+// Created by Lunar on 07/03/2022.
+//
+
+import Foundation
+import SwiftUI
+
+struct BottomCardView: View {
+    @StateObject var viewModel: BottomCardViewModel
+    private var onMarkerChangeAction: ((Int) -> ())? = nil
+    
+    init(id: Int, title: String, startTime: String, endTime: String, latitude: Double, longitude: Double) {
+        _viewModel = .init(wrappedValue: .init(id: id, title: title, startTime: startTime, endTime: endTime, latitude: latitude, longitude: longitude))
+    }
+    
+    var body: some View {
+        sessionCard
+    }
+    
+    var sessionCard: some View {
+        Button {
+            viewModel.setIsModalScreenPresented()
```

Comment:
As I understand, the function should be called more something like `sessionCardTapped` as the view tells the viewModel what happed rather than what it should do. Did I get it right or is it the other way around @pawelgil ?

Reply:
You right, my mistake